libs/aws/odc/aws/queue.py

```diff
@@ -87,37 +87,12 @@ def get_queue(queue_name: str):
     return queue
 
 
-def list_queues(region: Optional[str] = None):
+def list_queues():
```

Comment:
```suggestion
def get_queues():
```

Let's be consistent with the other one